modules/epp/iam_policies.tf

```diff
@@ -0,0 +1,117 @@
+resource "aws_iam_policy" "read_write" {
+  name        = "epp-${var.env}-s3-read-write"
+  path        = "/"
```

Comment:
Does this one policy allow read-write access to everything in the S3 bucket? If yes we should have different policies for the semi-automated process S3 stuff and import as to not have _any_ risk of overlapping buckets.

Reply:
import currently uses the root, so we'd need to change the import paths to have a prefix. That is trivial, but for the moment it's not possible to accommodate that if and until we change that in import.

under semi-automated, server always uses `data/`, but the server itself doesn't need anything but read access, which is why I'm creating the `image_server_role` that is read only. I will then switch over the process.

under-semi-automated, we rely on the developers to fill the bucket, and they probably should retain read/write to every path.